react/IconButton/styles.styl

```diff
@@ -1,30 +0,0 @@
-.IconButton
-    background none
-    border 0
-    cursor pointer
-    padding 0
-
-    display inline-flex
-    flex-direction column
-    justify-content center
-    align-items center
-
-    color var(--secondaryTextColor)
-
-    height 3rem
-    width 3rem
-
-    border-radius 3rem
-    box-sizing border-box
-
-    &:hover:not(:disabled)
-        background rgba(0, 0, 0, .06)
-
-    &:active:not(:disabled)
-        color var(--slateGrey)
-        background rgba(0, 0, 0, .12)
-        outline 0
-
-    &:disabled
-        opacity .24
-        cursor default
```

Comment:
So much code removed. Are we handling all these cases (hover / disabled / active) now?